src/state/vimState.ts

```diff
@@ -25,6 +24,7 @@ import { ReplaceState } from './../state/replaceState';
  */
 export class VimState implements vscode.Disposable {
   private readonly logger = Logger.get('VimState');
+  private _disposables: vscode.Disposable[] = [];
```

Comment:
Is this needed if nothing `VimState` holds is disposable (after you pulled `nvim` out)?